src/components/App/index.vue

```diff
@@ -87,7 +127,19 @@ export default {
 
 <style lang="scss" scoped>
 .App {
+  min-height: 100vh;
+
+  // FIXME: do this for iOS only
+  position: fixed;
```

Comment:
this is also supposed to be ios-only. you could throw a `.is-ios` classname on .App if it's being rendered on ios